cmd/tf-operator.v2/app/server.go

```diff
@@ -159,6 +161,23 @@ func Run(opt *options.ServerOption) error {
 }
 
 func createClientSets(config *restclientset.Config) (kubeclientset.Interface, kubeclientset.Interface, tfjobclientset.Interface, error) {
+
+	crdClient,err := crdclient.NewForConfig(config)
+	crdList, err := crdClient.ApiextensionsV1beta1().CustomResourceDefinitions().List(metav1.ListOptions{})
```

Comment:
I am wondering why we use list here. Could we follow the implementation as coreos/etcd-operator does https://github.com/coreos/etcd-operator/blob/c8f63d508266990a4d20718d94363c30e75e6282/pkg/util/k8sutil/crd.go#L79 ?